identity/app/model/SaveForLaterPageData.scala

```diff
@@ -2,30 +2,63 @@ package model
 
 import com.google.inject.{Inject, Singleton}
 import com.gu.facia.api.models.CollectionConfig
-import com.gu.identity.model.SavedArticles
-import common.Pagination
+import com.gu.identity.model.{SavedArticle, SavedArticles}
+import common.{ExecutionContexts, Edition, Pagination}
+import conf.LiveContentApi
+import conf.LiveContentApi._
 import layout.{ItemClasses, FaciaCard, ContentCard}
+import model.{Content => ApiContent}
 import services.{FaciaContentConvert, IdRequestParser, IdentityRequest, IdentityUrlBuilder}
 import implicits.Articles._
 import cards._
 
+import scala.concurrent.Future
+
+case class SaveForLaterItem(
+  content: Content,
+  savedArticle: SavedArticle) {
+
+  val contentCard = FaciaCard.fromTrail(
+    FaciaContentConvert.frontentContentToFaciaContent(content),
+    CollectionConfig.empty,
+    ItemClasses(mobile = cards.SavedForLater, tablet = cards.SavedForLater),
+    showSeriesAndBlogKickers = false
+  )
+}
+
+case class SaveForLaterPageData(
+  formActionUrl: String,
+  savedItems: List[SaveForLaterItem],
+  pagination: Pagination,
+  pageUrl: String,
+  totalArticlesSaved: Int,
+  shortUrls: List[String])
 
 @Singleton
```

Comment:
You better get used to this because from `play` `2.4`, dependency injection with `guice` is going to be the norm!